src/pathfinding_igvc/include/PathFinding.h

```diff
@@ -0,0 +1,54 @@
+//
+// Created by william on 20/01/18.
+//
+
+#ifndef PATHFINDING_IGVC_PATHFINDING_H
+#define PATHFINDING_IGVC_PATHFINDING_H
+
+#include "../../sb_utils/include/sb_utils.h"
```

Comment:
This can be cleaned up by referencing `sb_utils` in `CMakelists.txt` and `package.xml` and then instead:
```
#include "sb_utils.h"
```

Reply:
Done